src/app/components/challenge/challengephases/editphasemodal/editphasemodal.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="modal-container">
+  <div class="modal-card">
+    <div class="title">
+      {{title}}
+    </div>
+    <div class="form">
+      <div class="row row-lr-margin">
+        <div class="col-lg-6 col-md-6 col-sm-6 col-lr-pad">
+          <div class="input-field align-left">
+            <div class="fs-16 w-400 text-med-black">Phase Name</div>
+            <app-input class="mt-20" [isRequired]="true"
+              [editPhaseDetails]="editPhaseDetails" [label]="'name'"
+              [placeholder]="''" [type]="'text'" [icon]="" [value]="name"
+              #formmodal></app-input>
+          </div>
+        </div>
+      </div>
+
+      <div class="row row-lr-margin">
+        <div class="col-lg-12 col-md-12 col-sm-12 col-lr-pad">
+          <div class="input-field align-left">
+            <div class="fs-16 fw-regular text-med-black">Description</div>
+            <div class="editor-validation-message fs-12"
+              *ngIf="isEditorFieldMessage">{{editorValidationMessage}}</div>
+            <div class="mt-20 mb-20 edit-phase-editor" [froalaEditor]
+              [(froalaModel)]="description"></div>
+          </div>
+        </div>
+      </div>
+
+      <div class="row row-lr-margin"></div>
+
+      <div class="row row-lr-margin">
+        <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+          <div class="input-field align-left">
+            <span class="text-med-black fw-regular fs-16">Start date and time</span>
```

Comment:
Minor: Time in date-picker is in **24** Hr. format and what's displayed after selection is in **12** Hr. format. Should this be changed?

Reply:
thanks :)